workflows/config/snakemake/production_config_HomSap.yml

```diff
@@ -0,0 +1,55 @@
+"population_id": [0] # (what is this a setting for?)
+
+# snakemake settings
+"seed": 12345
+"replicates": 3
+"output_dir": "results"
+
+# species-specific settings
+"species": "HomSap"
+"demo_models": [
+    {"id":"Constant", 
+    "num_samples_per_population": [100],
+    },
+    {"id":"OutOfAfrica_3G09",
+    "num_samples_per_population": [100,100,100],
+    }
+    {"id":"OutOfAfricaArchaicAdmixture_5R19",
+    "num_samples_per_population": [100,100,100],
+    }
+]
+"genetic_map": "HapMapII_GRCh37"
```

Comment:
We should change this to "HapMapII_GRCh38" and update the mask_file below accordingly.